WooCommerce/UITestsFoundation/Screens/Orders/NewOrderScreen.swift

```diff
@@ -40,17 +68,28 @@ public final class NewOrderScreen: ScreenObject {
         return try SingleOrderScreen()
     }
 
+    /// Changes the new order status to the second status in the Order Status list.
+    /// - Returns: New Order screen object.
     @discardableResult
-    public func cancelOrderCreation() throws -> OrdersScreen {
-        cancelButton.tap()
-        return try OrdersScreen()
+    public func editOrderStatus() throws -> NewOrderScreen {
+      return try openOrderStatusScreen()
+            .selectOrderStatus(atIndex: 1)
```

Comment:
We could also update this so that status can be selected by type (instead of index), when we have that we can validate that the correct status is selected when the order is created (without worrying about the sort order changing in the future, which we have to think about if we use index).

Reply:
Since this test is intended to be minimal/functional, we didn't add checks for data. Our unit tests also check that the correct status is selected, so we decided we don't need to check the data at the UI level.